RELEASING.md

```diff
@@ -14,6 +14,28 @@ Stable releases are now managed by internal tooling (PCT)
 
 Use the instructions bellow as a fallback.
 
+## Generate the SBOM
```

Comment:
Can we also leave a link for this in https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=100255555#MongoDBAtlasCLI(AtlasCLI)andMongoDBCLI(mongocli)-ReleasingMongoCLIandAtlasCLI